src/initialize.js

```diff
@@ -204,9 +204,10 @@ export function loadExternalScripts(externalScripts, data) {
  */
 export function setCustomPrimaryColors() {
   const { CUSTOM_PRIMARY_COLORS } = getConfig();
+  const { PARAGON_THEME_URLS } = getConfig();
   const primary = CUSTOM_PRIMARY_COLORS['pgn-color-primary-base'];
 
-  if (!primary) {
+  if (!primary || PARAGON_THEME_URLS != null) {
```

Comment:
This make me thing... whatever you define with the var `PARAGON_THEME_URLS` in the mfe_config would discard the `CUSTOM_PRIMARY_COLORS` feature.  But for the moment seems functional.